autobahn/wamp_invocation.ipp

```diff
@@ -193,12 +199,13 @@ inline void wamp_invocation_impl::result(const List& arguments)
     message->set_field(3, arguments);
 
     m_send_result_fn(message);
-    m_send_result_fn = send_result_fn();
+    if (!intermediate)
+		m_send_result_fn = send_result_fn();
```

Comment:
Code elsewhere in autobahn-cpp is using braces for all `if` statements, including one-liners. That is good practice because it minimizes the potential for future errors when somebody might try to add a second line, and should really be adopted by all C/C++ developers everywhere. At least in this project though.